### PR TITLE
Replaced Godeps with glide

### PR DESCRIPTION
**Pull request by tboerger**

### CODE REVIEW

**Comment by jonboulle** [comment]
thanks for the PR! unfortunately still failing CI..

**Comment by tboerger** [comment]
> thanks for the PR! unfortunately still failing CI..

Yeah, and I found out why. You are still building with Go 1.4 -.-

**Comment by jonboulle** [comment]
whoops, that's awkward - we handle go a little differently in semaphore. Bumped to 1.6.2.

**Comment by tboerger** [comment]
I have rebased it on master and everything is green.

**Comment by s-urbaniak** [comment]
@tboerger thanks a lot for you PR! I am working on replacing godep with glide in rkt (https://github.com/coreos/rkt/pull/2735).

As more and more projects move to glide here at CoreOS, it would be beneficial to have one tool "that rules them all".

Do you mind to update your PR and use https://github.com/Masterminds/glide instead of govend?

**Comment by tboerger** [comment]
I personally don't really like glide and I prefer the simplicity of govend, but of course I can understand if you want to have glide, that's up to you :)

**Comment by tboerger** [comment]
@s-urbaniak I have updated the PR to use glide instead of govend. So now it's fine? :)

**Comment by s-urbaniak** [comment]
@tboerger Thanks! For the sake of reviewability do you mind splitting the introduction of https://github.com/appc/docker2aci/pull/174/commits/1343a412bde8213ef455546904077abefe2cf200 in two commits one including manual changes (introduction of glide, any other manual changes) and a separate commit for the actual vendored directory?

**Comment by tboerger** [comment]
> @tboerger Thanks! For the sake of reviewability do you mind splitting the introduction of 1343a41 in two commits one including manual changes (introduction of glide, any other manual changes) and a separate commit for the actual vendored directory?

So you want to have separate commits for adding glide.* and for dropping Godeps path from build script? The rust is just vendoring the libs.

**Comment by s-urbaniak** [comment]
@tboerger yes, in total three commits, 1) one for dropping godeps 2) one for adding the vendor dir and 3) one for manual changes (glide.yaml, scripts, et al), see https://github.com/coreos/rkt/pull/2735/commits

**Comment by tboerger** [comment]
@s-urbaniak I took more than 3 commits now, but it should be nicely grouped now.

**Comment by tboerger** [comment]
I have applied the same style of commit messages like on my other pull requests.

File: build.sh [hunk 1]
```diff
@@ -1,21 +1,33 @@
...
+if [ "${GOOS}" = "freebsd" ]; then
+    # /usr/bin/cc is clang on freebsd, but we need to tell it to go to
+    # make it generate proper flavour of code that doesn't emit
+    # warnings.
+    export CC=clang
+fi
+
 echo "Building docker2aci..."
-go build -o $GOBIN/docker2aci -ldflags "${GLDFLAGS}" ${REPO_PATH}/
+go build -o ${GOBIN}/docker2aci  -ldflags "${GLDFLAGS}" ${REPO_PATH}
```
Comment (iaguis) [inline build.sh]: Extra space, not that it matters much

Comment (tboerger) [inline build.sh]: Removed

**Comment by s-urbaniak** [comment]
@tboerger thanks, lgtm :+1:

**Pull request merged**